Tests/Functional/Controller/FrontEndEditorControllerTest.php

```diff
@@ -14,12 +15,16 @@
 final class FrontEndEditorControllerTest extends AbstractFrontendControllerTestCase
 {
     private const PAGE_UID = 1;
+    private const TEA_UID_ASSIGNED_TO_LOGGED_IN_USER = '1';
+    private const TEA_UID_ASSIGNED_TO_FOREIGN_USER = '2';
+    private const TEA_UID_ASSIGNED_TO_MISSING_USER = '3';
```

Comment:
I'd prefer the wording `OWNED_BY` (to be consistent with the wording of the property `owner`).

Reply:
(And for the last line: `TEA_UID_WIHOUT_OWNER`)